apidoc/Facebook.yml

```diff
@@ -227,134 +380,249 @@ description: |
     and [TIMODOPEN-391](https://jira.appcelerator.org/browse/TIMODOPEN-391).
 
 extends: Titanium.Module
-since: "3.5.1"
+since: "3.1.0"
 platforms: [android, iphone, ipad]
 methods:
+  - name: authorize
+    summary: |
+        Prompts the user to log in (if not already logged in) and authorize your application.
+        You can also use [LoginButton](Modules.Facebook.LoginButton) to log in.
+    description: |
+        Be sure to set your required [permissions](Modules.Facebook.permissions) before
+        calling `authorize`.
+
+        A [login](Modules.Facebook.login) event is generated to indicate a successful or
+        unsuccessful login attempt.
+        
+        #### iOS Platform Notes
+
+        On iOS, do not request any write permissions before calling this method.  Use the
+        [requestNewPublishPermissions()](Modules.Facebook.requestNewPublishPermissions)
+        to request write permissions once the user authorizes the application.
+
+        #### Prior to Release 4.0.0
+
+        Set the [appid](Modules.Facebook.appid) property as well as the `permissions` property
+        before calling `authorize()`.
+
+  - name: createActivityWorker
+    summary: |
+        Creates a Facebook proxy to hook into the activity of either a standalone <Titanium.UI.Window>
+        (not inside a TabGroup) or <Titaniuim.UI.TabGroup>.
+    description: |
+        Set the `lifecycleContainer` property in the dictionary passed to the method to either
+        the current active  instance of a <Titanium.UI.Window> or <Titanium.UI.TabGroup> in order to monitor the activity's
+        lifecycle events, required by Facebook to synchronize its state between various
+        activities in the application.
+
+        The proxy object must be created before calling the `open()` method on the associated Window
+        or TabGroup.
+    parameters:
+      - name: parameters
+        summary: |
+            Properties to set on a new object, including any defined by <Titanium.Proxy> except
+            those marked not-creation or read-only.
+
+            **Note:** You must set the `lifecycleContainer` property.
+        type: Dictionary<Titanium.Proxy>
+    returns:
+        type: Titanium.Proxy
+    since: 4.0.0
+    platforms: [android]
+
+  - name: dialog
```

Comment:
This is marked as deprecated and removed.  Still need to keep it around for people using older versions of the module that don't want to upgrade right away.